tests/TestHelpers/GraphHelper.php

```diff
@@ -499,43 +499,6 @@ public static function deleteGroup(
 		);
 	}
 
-	/**
-	 * @param string $baseUrl
-	 * @param string $xRequestId
-	 * @param string $adminUser
-	 * @param string $adminPassword
-	 * @param string $groupId
-	 * @param array $users expects users array with user ids
-	 *                     [ [ 'id' => 'some_id' ], ]
-	 *
-	 * @return ResponseInterface
-	 * @throws GuzzleException
-	 */
-	public static function addUsersToGroup(
```

Comment:
why deleting this function?

Reply:
- We don't have a single step that uses this method.
- we have no such endpoint in the list and the method does not work